packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -269,7 +268,7 @@ export const activate = async (extensionContext: vscode.ExtensionContext): Promi
     );
   }
 
-  telemetryService.sendExtensionActivationEvent(extensionHRStart);
+  telemetryService.sendExtensionActivationEvent();
```

Comment:
Some of these I was able to change such that just telemetry service gets the time. I want to expand that pattern since I don't think we're passing in anything aside from now() equivalents each time anyways.